### DISCUSSION TITLE
#4차 중간발표 [3월11일]

### DISCUSSION THREAD

**Discussion by cheol2Y** [jms0522/Medical-project#101] [Show and tell]
# 📗 4차 중간 발표 - 프로젝트 진행 상황

## 전체적인 작업 내용
- 전체 작업 결과물을 EC2로 이전
- 웹, 모니터링 시스템, 크롤링, 데이터베이스, 모델 구축
- 추가 서비스 구현
- docker-compose를 활용한 이전 과정

### 민수님
- **Monitoring System (Grafana, Prometheus)**
  - 진행 상황: [Monitoring 진행상황](https://github.com/jms0522/Medical-project/issues/77)
- **Neo4j를 이용한 Graph DB 구현** 
  - 진행상황 : #41 
  - 지식백과 데이터는 컬럼들이 연관성을 가지고 있기 때문에 그래프 형식으로 묶어서 벡터 DB 구현
  - 병명, 원인, 증상, 치료 등의 데이터를 그래프 형식으로 구성하여 정확성 및 속도 향상 예상
  - 관련 데이터: [지식백과 데이터](https://terms.naver.com/entry.naver?docId=926907&cid=51007&categoryId=51007)

### 민석님
- **웹 이식 (EC2, Docker-compose 미적용) 및 코드 최적화**
  - 진행 상황: [Web 진행 상황](https://github.com/jms0522/Medical-project/issues/64)

### 승표님
- **서비스 로직 구체화 및 구현**
  - 벡터 DB에 저장된 데이터 중 유사한 데이터 5개 출력
  - 진행 상황: [추가 서비스 진행상황](https://github.com/jms0522/Medical-project/issues/95)
- **모델 프롬프트 튜닝**

### 성철님
- **Docker-compose 변경사항 적용**
  - 진행 상황: [Docker 진행 상황](https://hub.docker.com/repository/docker/cheol2y/custom_airflow/general)
- **Data 크롤링 변경사항 적용 및 Data pipeline 구축**

### 멘토링
- **문서 작성 및 작업 방향성 제시**

### 민수님
- **ELK stack을 활용한 log 모니터링 시스템 구축**
- **Docker-compose로 구현**
- **모델 확장**

### 민석님
- **유사답변 출력 서비스 추가**
- **ChromaDB 연결 로직 변경**
- **코드 최적화**

### 승표님
- **서비스 로직 추가**
- **추가한 서비스 Docker-compose로 구현**

### 성철님
- **Crawling Data 파이프라인 완벽 구축**
- **문서 작업**

**Comment by jms0522** [comment]
본좌 이러네 🤨
